docs/contributing/templates/c2c-source.md

```diff
@@ -0,0 +1,103 @@
+1. **Collecting**. The Source is actively collecting data from Microsoft.
+
+If a Source encounters an issue during any of the above states, it is placed in an **Error** state. When you delete a Source, it is placed in a **Stopping** state. When it has successfully stopped, it is deleted from your Hosted Collector.
+
+### Health Events
+
+To view Health Events and Status for your Source, go to **Manage** > **Collection**> **Collection**. Under the Health column, you can click an **Error** status to open the  Health Events and investigate issues that may have occurred during collection.
+
+![hover-c2c-error.png](/img/send-data/hover-c2c-error.png)
+
+After you click on **Error**, you should see a **Health Events** popup modal containing all detected errors and warnings. Next, click the three-dot icon > **View Details** to open a Health Events panel with details on each detected issue.
+
+## Setup and configuration
+
+{{Configure source in Sumo Logic UI}} <!-- Example: https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/ms-graph-identity-protection-source/#create-amicrosoft-graph-identity-protection-source -->
+
+{{Configure on source's website (not required for all sources)}} <!-- Example: https://help.sumologic.com/docs/send-data/hosted-collectors/cloud-to-cloud-integration-framework/ms-graph-identity-protection-source/#set-up-a-service-application -->
+
+## Error Types
+
+When Sumo Logic detects an issue, it is tracked by Health Events. The following table shows the possible error types, the reason for the error, if the source attempts to retry, and the event log name in the Health Event Index.
+
+| Type | Reason | Retries | Retry Behavior | Health Event Name |
+|:--|:--|:--|:--|:--|
+| ThirdPartyConfig  | Normally due to an invalid configuration. You'll need to review your Source configuration and make an update. | No retries are attempted until the Source is updated. | Not applicable | ThirdPartyConfigError  |
+| ThirdPartyGeneric | Normally due to an error communicating with the third-party service APIs. | Yes | The Source will retry indefinitely. | ThirdPartyGenericError |
+| FirstPartyGeneric | Normally due to an error communicating with the internal Sumo Logic APIs. | Yes | The Source will retry indefinitely. | FirstPartyGenericError |
+
+### Restarting your Source
+
+<!--
+Paste this line: {@import ../../reuse/restart-c2c-source.md}
+-->
+
+## JSON configuration
+
+Sources can be configured using UTF-8 encoded JSON files with the [Collector Management API](/docs/api/collector-management). See [Using JSON to Configure Sources](/docs/send-data/use-json-configure-sources) for details. 
+
+| Parameter | Type | Required | Description | Access |
+|:--|:--|:--|:--|:--|
+| config | JSON Object  | Yes | Contains the configuration parameters for the Source. |   |
+| schemaRef | JSON Object  | Yes | Set to `{"type":"MS Graph Azure AD Reporting"}`. | not modifiable |
```

Comment:
```suggestion
| schemaRef | JSON Object  | Yes | Set to `{"type":"Source name"}`. | not modifiable |
```

Reply:
Ah, I forgot to remove these - thank you!